workflows/clean-era5-workflow.yaml

```diff
@@ -11,9 +11,9 @@ spec:
   arguments:
     parameters:
     - name: in-zarr
-      value: "clean-dev/ERA-5/tmax.1995-2014.0p25.zarr"
+      value: "clean-dev/ERA-5/F320/dtr.1995-2015.F320.v2.zarr"
```

Comment:
Ehhh. What do you think about swapping this default value to point to the new tasmax zarr store, instead of dtr? 

I suggest this because the entire pipeline has been gear to tasmax up to this point. We _should_ get dtr and the other variables, in addition, but I'd argue that tasmax is the key right now while we're putting this together.